.idea/encodings.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="Encoding" addBOMForNewFiles="with NO BOM" />
```

Comment:
We shouldn't be changing any `.idea` files as part of this PR.  So these changes should be removed

Reply:
to be honest, I don't know what these changes are nor how I triggered them